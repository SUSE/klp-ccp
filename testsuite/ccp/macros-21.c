#include "macros-21.h"

#define FOO void

FOO pu_f(void)
{
	g();
}
