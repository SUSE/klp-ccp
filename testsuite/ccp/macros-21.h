
#define FOO int

FOO g(void)
{
	return 0;
}

#undef FOO
