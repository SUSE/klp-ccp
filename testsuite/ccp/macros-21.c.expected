/* klp-ccp: from testsuite/ccp/macros-21.h */
#define FOO int

static FOO g(void);

/* klp-ccp: from testsuite/ccp/macros-21.c */
#include "macros-21.h"

#define FOO void

FOO klpp_pu_f(void)
{
	g();
}
